mmv1/products/redis/api.yaml

```diff
@@ -143,6 +143,125 @@ objects:
           The ID of the instance or a fully qualified identifier for the instance.
         required: true
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: maintenancePolicy
+        description: Maintenance policy for an instance.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'createTime'
+            output: true
+            description: |
+              Output only. The time when the policy was created.
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits.
+          - !ruby/object:Api::Type::String
+            name: 'updateTime'
+            output: true
+            description: |
+              Output only. The time when the policy was last updated.
+              A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+              resolution and up to nine fractional digits.
```

Comment:
should this field be included? It does not seem useful for someone creating a terraform configuration?

Reply:
Well, it's a part of API response, and I wanted to map it exactly - this will be stored in state allowing users to read this as attribute and maybe do something based on this. I see no downside of having it.
Please let me know if that's ok